src/index.d.ts

```diff
@@ -952,6 +960,11 @@ declare namespace admin.auth {
      * The buffer of bytes representing the user’s password salt.
      */
     passwordSalt?: Buffer;
+  
+    /**
+     * The identifier of the tenant where user is to be imported to.
+     * When not provided, the user is uploaded to the default parent project.
```

Comment:
If not provided, the user is uploaded to the tenant specified in the request parameter `tenantId`

Reply:
I expanded the description. Please take another look.